apps/src/music/player/MusicPlayer.ts

```diff
@@ -285,9 +286,11 @@ export default class MusicPlayer {
         return [];
       }
 
+      const url = library.generateSoundUrl(folder, soundData);
+
       return [
         {
-          sampleId: `${folder.path}/${soundData.src}`,
+          sampleId: url,
```

Comment:
might be good to just change this field name to `sampleUrl` or `samplePath` for clarity?

Reply:
Done.